.github/DISCUSSION_TEMPLATE/question.yml

```
@@ -0,0 +1,63 @@
+title: "Summary"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We are glad to have you build on Aptos with us!
+        Please choose some labels and fill out the questions below.
+  - type: textarea
+    id: question
+    attributes:
+      label: Describe your question in detail
+    validations:
+      required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: What error are you getting?
+    validations:
+      required: true
```

This should be not required, sometimes there's no error.

```suggestion
  - type: textarea
    id: error
    attributes:
      label: What error, if any, are you getting?
    description: |
      If no error, you can leave this field blank.
    validations:
      required: false
```